### PR TITLE
Feat: client exp display

### PR DESCRIPTION
**Pull request by omarcopires**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed 
Now send rate as expected instead of always sending a fixed value to the player.

image

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: src/player.h
Comment (jo3bingham) [inline src/player.h:1116]:
```suggestion
		baseExpGain = value;
```
`std::min` is pointless as `value` can't exceed the max value of `uint16_t` because it is a `uint16_t`.

Comment (omarcopires) [reply src/player.h]: I have almost no knowledge of C, thanks for the help.

**Review by jo3bingham — APPROVED** [review]
This LGTM, but you may need approval from someone on the dev team before it can be merged.

**Review by EPuncker — DISMISSED** [review]
if jo3s agree, I agree as well :)

File: src/player.h
Comment (marmichalski) [inline src/player.h:1261]: This is misleading as this does not affect exp gain rate at all and is only used for display purposes

Comment (marmichalski) [reply src/player.h:1261]: This is in no way related to players baseExpGain, if you set it to 200 player won't start getting more XP. All it does is update value in Client (display).

Calling player:setBaseExpGain does not affect the exp gain.

Comment (omarcopires) [reply src/player.h:1261]: any name suggestions?

**Comment by omarcopires** [comment]
I can't think of better names for the functions, this is the best I could come up with, hope that's good enough.

**Review by EPuncker — DISMISSED** [review]

**Review by EPuncker — APPROVED** [review]

**Review by DSpeichert — APPROVED** [review]

**Pull request merged**
